maint/extractcvars

```diff
@@ -252,6 +252,10 @@ int ${fn_ns}_init(void)
     MPIR_T_cvar_value_t defaultval;
 
     int debug = 0;
+    rc = MPL_env2bool("MPICH_DEBUG_CVARS", &debug);
+    MPIR_ERR_CHKANDJUMP1((-1 == rc),mpi_errno,MPI_ERR_OTHER,"**envvarparse","**envvarparse %s","MPICH_DEBUG_CVARS");
+    rc = MPL_env2bool("MPIR_PARAM_DEBUG_CVARS", &debug);
+    MPIR_ERR_CHKANDJUMP1((-1 == rc),mpi_errno,MPI_ERR_OTHER,"**envvarparse","**envvarparse %s","MPIR_PARAM_DEBUG_CVARS");
```

Comment:
I'm not sure how these are related to the other code changes

Reply:
They are not. I noticed that we only checked `MPIR_CVAR_DEBUG_CVARS`, while all other cvars can also use `MPICH_` and `MPIR_PARAM_` suffixes, so I patch it for consistency.